src/reviews.rs

```diff
@@ -119,3 +123,26 @@ pub(crate) async fn delete_review(
 
   Ok(())
 }
+
+async fn validate_instructors(
+  db: Arc<Db>,
+  course_id: &str,
+  instructors: &[String],
+) -> Result {
+  let course = db
+    .find_course_by_id(course_id)
+    .await?
+    .ok_or(anyhow!("Invalid course"))?;
+
+  let valid_instructors: Vec<String> =
+    course.instructors.into_iter().map(|ins| ins.name).collect();
```

Comment:
```suggestion
  let valid_instructors =
    course.instructors.into_iter().map(|ins| ins.name).collect::<Vec<String>>();
```

Super small nit, but i think turbofish reads nicer